http://sourceware.org/ml/gdb-cvs/2012-09/msg00132.html

### src/gdb/ChangeLog	2012/09/22 13:04:54	1.14691
### src/gdb/ChangeLog	2012/09/24 10:25:07	1.14692
## -1,3 +1,8 @@
+2012-09-24  Siddhesh Poyarekar  <siddhesh@redhat.com>
+
+	* m2-typeprint.c (m2_enum): Expand LASTVAL to LONGEST.
+	* p-valprint.c (pascal_type_print_base): Likewise.
+
 2012-09-22  Yao Qi  <yao@codesourcery.com>
 
 	* remote.c (remote_get_trace_status): Remove setting default
--- src/gdb/m2-typeprint.c	2012/04/18 06:46:46	1.30
+++ src/gdb/m2-typeprint.c	2012/09/24 10:25:09	1.31
@@ -587,7 +587,8 @@
 void
 m2_enum (struct type *type, struct ui_file *stream, int show, int level)
 {
-  int lastval, i, len;
+  LONGEST lastval;
+  int i, len;
 
   if (show < 0)
     {
--- src/gdb/p-typeprint.c	2012/08/16 07:36:20	1.47
+++ src/gdb/p-typeprint.c	2012/09/24 10:25:09	1.48
@@ -440,7 +440,7 @@
 {
   int i;
   int len;
-  int lastval;
+  LONGEST lastval;
   enum
     {
       s_none, s_public, s_private, s_protected
